src/acom_music_box/data_output.py

```diff
@@ -0,0 +1,65 @@
+import os
+import pandas as pd
+import xarray as xr
+
+class DataOutput:
+    def __init__(self, df, args):
+        self.df = df
+        self.args = args
+        self.unit_mapping = {
+            'ENV.temperature': 'K',
+            'ENV.pressure': 'Pa',
+            'ENV.number_density_air': 'kg -m3',
+            'time': 's'
+        }
```

Comment:
Please add a docstring to the `DataOutput` class describing its purpose and use. Also, please add a docstring to `__init__` describing the arguments. I don't think we've standardized our documentation yet, but we should strive for [numpy-style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html). If you notice, those can also include examples in the comments. Please attempt to add an example usage in the docstring for `DataOutput`